sdv/metadata/single_table.py

```diff
@@ -300,6 +300,28 @@ def update_column(self, column_name, **kwargs):
         self.columns[column_name] = _kwargs
         self._updated = True
 
+    def update_columns(self, column_names, **kwargs):
+        """Update multiple columns with the same metadata kwargs.
+
+        Args:
+            column_names (list[str]):
+                A list of column names to be updated.
+            **kwargs (type):
+                Any key word arguments that describe metadata for the column.
+        """
+        for column_name in column_names:
+            self.update_column(column_name, **kwargs)
+
+    def update_columns_metadata(self, column_metadata):
+        """Update the metadata for multiple columns using metadata from the input dictionary.
+
+        Args:
+            column_metadata (dict):
+                A dictionary of column names and their metadata to be updated.
+        """
+        for column_name, metadata in column_metadata.items():
+            self.update_column(column_name, **metadata)
```

Comment:
Same thing, would it be better for this to be in a try/catch instead? In other words, should we try to update all of the columns and then return errors for any that weren't successfully updated?